### PR TITLE
remove nRecordPoints in all examples

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**

### CODE REVIEW

File: convergence_elastic/parameters2.par [hunk 1]
```diff
@@ -0,0 +1,61 @@
+&Equations
+rho = 1.
+mu = 1.
```
Comment (krenzland) [inline convergence_elastic/parameters2.par:3]: What's the purpose of this file? Did you commit this by accident?

**Review by krenzland — CHANGES_REQUESTED** [review]
Great, thanks. Everything looks perfect, only " convergence_elastic/parameters2.par " is a bit confusing? Did you commit the wrong file?

File: convergence_elastic/parameters2.par (hunk 1 above)
Comment (Thomas-Ulrich) [reply convergence_elastic/parameters2.par:3]: probably yes

Comment (Thomas-Ulrich) [reply convergence_elastic/parameters2.par]: ok that should be fixed now

**Review by krenzland — APPROVED** [review]
Thanks!

**Pull request merged**